packages/engine/src/worker/mod.rs

```diff
@@ -484,12 +496,40 @@ impl WorkerController {
         &mut self,
         sim_id: Option<SimulationShortId>,
         sync_msg: SyncPayload,
+        pending_syncs: &mut FuturesUnordered<Pin<Box<dyn Future<Output = ()> + Send>>>,
     ) -> Result<()> {
+        let sync = if let SyncPayload::State(sync) = sync_msg {
+            sync
+        } else {
+            tokio::try_join!(
+                self.py
+                    .send_if_spawned(sim_id, sync_msg.try_clone()?.into()),
+                self.js
+                    .send_if_spawned(sim_id, sync_msg.try_clone()?.into()),
+                self.rs.send_if_spawned(sim_id, sync_msg.into())
+            )?;
+            return Ok(());
+        };
+
+        // TODO: Change to `children(3)` after enabling all runners.
```

Comment:
Slightly worried we'll miss this. Any way we can code it in quickly or not really easily?

Reply:
I added an assert